.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: python
+os:
+  - linux
+python:
+  - "2.7"
+  - "3.5"
+  - "3.6"
+  - "pypy"
+install:
+  - "pip install flake8"
```

Comment:
flake8 is [a wrapper for pyflakes, pycodestyle, and Ned Batchelder’s McCabe script](https://pypi.org/project/flake8/).

Since flake8 isn't required for users who aren't developing, I don't think it is necessary to put it into requirements.txt. One thing I have seen others do is create a separate requirements-dev.txt file which is used for development. I can create this and read directly from that if you would like.

Reply:
```suggestion
  - "pip install -r requirements-dev.txt"
```

Reply:
Make sense but the project is small and so thought it would be okay rather than creating different dependency files.

Reply:
Ok, I will add it to requirements.txt.